drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -92,15 +92,28 @@ local function _open_coordinator_socket(sonos_conn, household_id, self_player_id
   end
```

Comment:
Should we have an early return here?

Reply:
Yeah, I updated the others, so I should probably update this one too. I dont think it makes a difference functionally though.